.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,12 +2,12 @@
 documentation before submitting a Pull Request!
 Thank you for contributing to Rook! -->
 
-Description of your changes:
+**Description of your changes:**
 
-Which issue is resolved by this Pull Request:
+**Which issue is resolved by this Pull Request:**
 Resolves #
 
-Checklist:
+**Checklist:**
 - [ ] Documentation has been updated, if necessary.
```

Comment:
Also a nit, but I'd make these checklist items less wordy. Something like:

**Checklist:**
If necessary:
- [ ] Documentation updated.
- [ ] Pending release notes updated.
- [ ] Upgrade tested.


I don't see a need for all of the extras. For instance, the user guide is documentation so you don't need to mention that again. It should also be obvious to any competent developer that the pending release notes are updated with any notable changes or changes that would break other features.

Reply:
while we could certainly have "if necessary" at the top, i do like the existing wording on these items. While they may seem obvious, i find myself getting reminded by these when opening a PR. Docs, release notes, and upgrades aren't always on a dev's mind.